### PR TITLE
Add Via porting and Stacked Acrylic Case

### PR DESCRIPTION
**Pull request by TahuTech**
hello, i love this yampad, i took the initiative to contribute to developing the case and also firmware.
i just add via porting for ez remapping for yampad, and add file stacked acrylic case.

### CODE REVIEW

**Review by mattdibi — CHANGES_REQUESTED** [review]
Hi @TahuTech 

thanks for your contribution. The stacked acrylic case looks awesome!

A note about the case files: I noticed you shared a `.cdr` file which is [proprietary](https://i.redd.it/7ozal346p6kz.png) and can be only opened through CorelDRAW. Could you share a few alternative formats for easier customization? (`.dxf` comes to mind but anything more open will do).

File: README.md [hunk 1]
```diff
@@ -169,6 +170,21 @@ See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_to
...
+
+<p align="center">
+<img src="img/acrylic_case/yampad (2 of 10).jpg" alt="Yampad Acrylic Case" width="300"/>
+<img src="img/acrylic_case/yampad (4 of 10).jpg" alt="Yampad Acrylic Case" width="300"/>
+</p>
+
+<p align="center">
+<img src="img/acrylic_case/yampad (10 of 10).jpg" alt="Yampad Acrylic Case" width="300"/>
+</p>
+
```
Comment (mattdibi) [inline README.md:187]: Add a link to the gallery of images.

```suggestion
Acrylic case image gallery [here](https://github.com/mattdibi/yampad/tree/master/img/acrylic_case/).
```

**Comment by TahuTech** [comment]
okay done convert to .dfx format

**Review by TahuTech — COMMENTED** [review]
change format case to .dfx

**Review by mattdibi — APPROVED** [review]
LGTM Thanks :)

**Pull request merged**